content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,92 @@
+- **Improved** load balancing to direct traffic to the [same region and zone](docs/ops/traffic-management/locality-load-balancing/) by default.
+- **Improved** Pilot performance. The reduction in CPU utilization could be up to 10 times depending on deployment.
+- **Improved** the `ServiceEntry` API to allow for the same hostname in different namespaces.
+- **Improved** the [Sidecar API](/docs/reference/config/networking/v1alpha3/sidecar/#OutboundTrafficPolicy) to customize the `OutboundTrafficPolicy` policy.
+
+## Security
+
+- **Added** a unified logging pattern to Citadel agent.
+- **Added** metrics to the Citadel agent to monitor SDS push.
+- **Improved** the ingress gateway to provide client CA certificates when using SDS.
+- **Added** trust domain validation for Envoy proxies using mutual TLS. Improved validation only applies to the `Sidecars` resource.
+- **Added** the `ca.istio.io/env` and `ca.istio.io/override` labels for operators to control service account secret generation by namespace.
+- **Removed** integration with Vault CA temporarily. SDS requirements caused the temporary removal but we will reintroduce Vault CA integration a future release.
+- **Added** SDS support to improve the performance of Envoy proxies handling secrets.
+- **Enabled** the Envoy JWT filter by default to improve security and reliability.
+- **Improved** SDS reliability with the requirement for Kubernetes 1.13. Note that Istio 1.3 still works on Kubernetes 1.12, but SDS [requires Kubernetes 1.13](/blog/2019/trustworthy-jwt-sds).
+
+## Telemetry
+
+- **Added** Access Log Service [ALS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/service/accesslog/v2/als.proto#grpc-access-log-service-als) support for Envoy gRPC.
+- **Added** a Grafana dashboard for Citadel monitoring.
+- **Added** [metrics](/reference/commands/sidecar-injector/#metrics) for monitoring the sidecar injector webhook.
+- **Added** control plane metrics to monitor Istio's configuration state.
+- **Added** telemetry reporting for traffic destined to the `Passthrough` and `BlackHole` clusters.
+- **Added** alpha support for in-proxy generation of service metrics using Prometheus.
+- **Added** alpha support for environmental metadata in Envoy node metadata.
+- **Added** alpha support for Proxy Metadata Exchange.
+- **Added** alpha support for the OpenCensus trace driver.
+- **Improved** the mesh dashboard to provide monitoring of Istio's configuration state.
+- **Improved** the Pilot dashboard to expose additional key metrics to more clearly identify errors.
+- **Removed** deprecated `Adapter` and `Template` custom resource definitions (CRDs).
+- **Deprecated** Mixer adapters. Please begin the transition of all extensions to the new extension model. Legacy Mixer integration support continues only for Istio 1.3 and 1.4.
```

Comment:
any link to point to how to transition?

Reply:
https://github.com/istio/istio.io/pull/4934#discussion_r322363689

Reply:
I think we should remove this from here.
We will announce deprecation separately in the workgroup, along with the transition doc.

Reply:
I feel like deprecation notices belong in the release notes. It seems important to put a stake in the ground that declares Mixer is deprecated as an architectural component. Even if the next sentence is "Details on how to transition existing OOP adapters, etc., will be published shortly".

Reply:
I think it is incorrect to mark deprecation without providing a path.  How can user begin transition?